build.gradle

```diff
@@ -17,12 +17,12 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.1'
```

Comment:
GSON has better kotlin support, so I chose to use that instead of adding Jackson's kotlin module (which broke kotlin code for some reason)